prime-angular-frontend/src/app/modules/site-registration/site-registration-routing.module.ts

```diff
@@ -46,6 +48,16 @@ const routes: Routes = [
         component: CollectionNoticePageComponent,
         data: { title: 'Collection Notice' }
       },
+      {
+        path: SiteRoutes.ORGANIZATION_CLAIM,
```

Comment:
With this relating to an organization this might be better located within the organization route loop.